CautionaryAlertsApi/V1/Boundary/Response/DiscretionAlertResponse.cs

```diff
@@ -0,0 +1,20 @@
+namespace CautionaryAlertsApi.V1.Boundary.Response
+{
+    public class DiscretionAlertResponse
```

Comment:
Hey Vlad - could we please call this CautionaryAlertResponse? I know this is the FE terminology, but in terms of data, those are still called Cautionary Alerts.

Reply:
There is already a [CautionaryAlertResponse](https://github.com/LBHackney-IT/cautionary-alerts-api/blob/master/CautionaryAlertsApi/V1/Boundary/Response/CautionaryAlertResponse.cs) (and [CautionaryAlertPersonResponse](https://github.com/LBHackney-IT/cautionary-alerts-api/blob/master/CautionaryAlertsApi/V1/Boundary/Response/CautionaryAlertPersonResponse.cs)) having different structure. It's possible to amend them with Type and PersonId fields, but I'm not sure it's the right way to do.

So we need to use another name. What about CautionaryAlertGoogleSheetResponse (or something like this)?

Reply:
Yes, that sounds fine (CautionaryAlertGoogleSheetResponse) - I just don't want to keep it as DiscretionaryAlert as someone might think it's a different type of data.

Reply:
Updated, no more discretion here